.github/workflows/int_test.yml

```diff
@@ -348,6 +358,25 @@ jobs:
           az iot ops check --as-object
           az iot ops check --svc broker --resources broker brokerlistener
           az iot ops asset query -g ${{ env.RESOURCE_GROUP }} --location westus -o table
+      - name: "Keep cluster alive"
+        if: ${{ inputs.keep-on-failure > 0 && failure() }}
```

Comment:
Can we store inputs.keep-on-failure to an intermediate env var, and access that? We should do the same for other inputs.

Reply:
done - also tried to do this for the runtime args too (discussed that the matrix still needs the inputs)